pkg/cmd/server/origin/master.go

```diff
@@ -120,6 +120,20 @@ func (fn APIInstallFunc) InstallAPI(container *restful.Container) []string {
 	return fn(container)
 }
 
+// assetServerOffNotice returns a notice that the web console is off if
+// this feature is disabled.
+func assetServerOffNotice(handler http.Handler) http.Handler {
```

Comment:
we don't want to register handlers for negative capabilities... if the /console endpoint isn't handled, you'll just get a normal server error